docker/server/build.gradle

```diff
@@ -0,0 +1,43 @@
+import io.deephaven.tools.License
+
+plugins {
+    id 'com.bmuschko.docker-remote-api'
+    id 'base'
+}
+
+evaluationDependsOn ':docker-runtime-base'
+evaluationDependsOn ':grpc-api-server-native'
+
+def dockerContext = project.layout.buildDirectory.dir('context')
+
+def dockerLicenses = License.createFrom(project).syncDockerLicense().get()
+
+def prepareDocker = project.tasks.register('prepareDocker', Sync) {
+    def distTar = project.project(':grpc-api-server-native').tasks.findByName('distTar')
```

Comment:
instead of task output, we can just depend on it as an artifact in the dependencies {} block, and skip the evaluationDependsOn - we don't care about the _task_, we care about the artifact.

Reply:
I think I've got that working now.